tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.js

```diff
@@ -0,0 +1,65 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+/**
+ * Theme & Logo base page, contains functions that can be used on the page
+ * @class
+ * @type {themeAndLogoBasePage}
+ * @extends BOBasePage
+ */
+module.exports = class themeAndLogoBasePage extends BOBasePage {
+  /**
+   * @constructs
+   * Setting up texts and selectors to use on theme & logo page
+   */
+  constructor() {
+    super();
+    this.advancedCustomizationNavItemLink = '#subtab-AdminPsThemeCustoAdvanced';
+    this.themeShopCard = '#themes-logo-page .card-header[data-role="theme-shop"]';
+    this.cardInactiveTheme = '#themes-logo-page .card-body :nth-child(2) .theme-card';
```

Comment:
This is not a solid a it seems, could we make better by introducing some `data-role` selectors ?